### PR TITLE
Update the troubleshoot section

### PR DESCRIPTION
**Pull request by mervynmanilall**
Dear team,
In this PR, I did the following changes:

- Added some information on how to tackle authenticate errors for systems which are not Windows.
- Added a section in the troubleshoot for error 403.

Regards

### CODE REVIEW

**Comment by Court72** [comment]
@ramiMSFT

Can you review the proposed changes? 

When the changes are ready for publication, add a `#sign-off` comment to signal that the PR is ready for the review team to merge. 

#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

**Comment by mervynmanilall** [comment]
Hello @ramiMSFT ,

Do you have any update please?

**Review by ramiMSFT — APPROVED** [review]

**Comment by ramiMSFT** [comment]
@mervynmanilall Thanks for your contribution! I've made some changes and the review team will merge this shortly.

#sign-off

File: docs/artifacts/npm/npmrc.md [hunk 1]
```diff
@@ -401,6 +401,10 @@ Follow these steps to reset your vsts-npm-auth credentials:
     npm install -g vsts-npm-auth --registry https://registry.npmjs.com --always-auth false
     ```
 
+#### Unable to pushlish
```
Comment (Jak-MS) [inline docs/artifacts/npm/npmrc.md:404]:
```suggestion
#### Unable to publish
```

**Pull request merged**